lib/reek/code_comment.rb

```diff
@@ -20,19 +21,42 @@ class CodeComment
     DISABLE_DETECTOR_CONFIGURATION = '{ enabled: false }'.freeze
     MINIMUM_CONTENT_LENGTH         = 2
     LEGACY_SEPARATOR               = ':'.freeze
+    UNKNOWN_SMELL_DETECTOR_MESSAGE = <<-EOS.freeze
```

Comment:
Fair enough - thanks for the tip! I was indeed not aware of this ;)
I'll keep this in mind for the future. If we introduce this syntax into Reek we probably should update all heredocs to be consistent from the start, which would warrant its own PR.